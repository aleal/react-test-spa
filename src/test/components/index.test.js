import React from 'react';
import {renderComponent} from '../test-helper';
import Index from '../../components/index';

describe('App Index Tests' , () => {
    
    let component;

    beforeEach(() => {
        component = renderComponent(Index);
    });
  
    it('renders index page', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});