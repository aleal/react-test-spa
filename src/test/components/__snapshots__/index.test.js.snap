// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Index Tests renders index page 1`] = `
<div
  className="container"
>
  <h3>
    Index
  </h3>
  <p>
     Welcome! Sign in or Sign up. 
  </p>
</div>
`;
